pallets/capacity/src/lib.rs

```diff
@@ -862,6 +865,9 @@ impl<T: Config> Pallet<T> {
 			UnstakeUnlocks::<T>::set(staker, Some(unlocks));
 		}
 
+		// Set the total locked amount. We do not simply apply the delta of the withdrawn chunks, as this
+		// is error-prone due to the double-locking issue; instead, we simply calculate the total locked
+		// amount as the `total staked + total remaining unthawed unlocking chunks`.
```

Comment:
I'm a bit confused about the location of this comment. Is this comment related to the changes for this PR or just a clarification of the logic which was implemented previously?

Reply:
It was a clarification comment that I added, more for my own benefit. Can remove if confusing.